packages/interactor/src/integration/cypress.ts

```diff
@@ -0,0 +1,38 @@
+import { bigtestGlobals, RunnerState } from '@bigtest/globals';
+import { Interaction, ReadonlyInteraction } from '../interaction';
+
+function interact(
+  interaction: Interaction<void> | ReadonlyInteraction<void>,
+  runnerState: RunnerState
+) {
+  bigtestGlobals.runnerState = runnerState;
+  return cy.document({ log: false }).then((doc: Document) => {
+    bigtestGlobals.document = doc;
+    return interaction;
+  }).then(() => {
+    Cypress.log({
+      displayName: runnerState,
+      message: interaction.description
+    });
+  })
+}
+
+Cypress.Commands.add('do', (
+  interaction: Interaction<void>
```

Comment:
@dagda1 Does this one look okay? The command can take either a single interactor or an array of interactors so I feel like I'm not typing this accurately.